source/MaterialXGenShader/ShaderGraph.cpp

```diff
@@ -251,13 +251,12 @@ void ShaderGraph::addDefaultGeomNode(ShaderInput* input, const GeomPropDef& geom
 void ShaderGraph::addColorTransformNode(ShaderInput* input, const ColorSpaceTransform& transform, GenContext& context)
 {
     ColorManagementSystemPtr colorManagementSystem = context.getShaderGenerator().getColorManagementSystem();
-    if (!colorManagementSystem || input->getConnection())
+    if (!input->getIsBindInput() && (!colorManagementSystem || input->getConnection()))
     {
-        // Ignore inputs with connections as they are not 
+        // Ignore inputs (that are not bind inputs or bind params with connections as they are not 
```

Comment:
Just missing a closing parenthesis here!  I might also recommend phrasing this comment "Ignore unbound inputs with connections, as they are not allowed to have colorspaces specified."  That covers both the bindinput and bindparam cases without the need for parentheses.